### PR TITLE
[tests-only] Add method setLastPublicSharefor ocis

### PR DESCRIPTION
**Pull request by v-scharf**
to using some gerkin steps from core in ocis I had to add the method setLastPublicSharefor because I could not set and get  `lastPublicShareData` `private` $lastPublicShareData`

### CODE REVIEW

**Review by phil-davis — COMMENTED** [review]
Do yoyu also want to

**Comment by ScharfViktor** [comment]
> Do yoyu also want to

Does anyone want to add also a same method?

**Comment by phil-davis** [comment]
@ScharfViktor sorry, I started typing in my review comment, then looked to see that `getLastPublicShareData` and `resetLastPublicShareData` methods already exist.

**Review by phil-davis — APPROVED** [review]

File: tests/acceptance/features/bootstrap/Sharing.php [hunk 1]
```diff
@@ -139,6 +139,13 @@ public function getLastPublicShareData():?SimpleXMLElement {
 		return $this->lastPublicShareData;
 	}
 
+	/**
+	 * @param SimpleXMLElement $responseXml
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Sharing.php:143]: https://drone.owncloud.com/owncloud/core/36378/3/7

code-style check wants a `@return void` in this PHPdoc comment

Comment (ScharfViktor) [reply tests/acceptance/features/bootstrap/Sharing.php:143]: I fixed it. but when I run `make test-php-style` I get: ```
PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 6295552 bytes) in /Users/scharfviktor/Work/core/vendor-bin/php_codesniffer/vendor/squizlabs/php_codesniffer/src/Util/Cache.php on line 291

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 6295552 bytes) in /Users/scharfviktor/Work/core/vendor-bin/php_codesniffer/vendor/squizlabs/php_codesniffer/src/Util/Cache.php on line 291
make: *** [test-php-style] Error 255
```

**Review by phil-davis — APPROVED** [review]

**Comment by ScharfViktor** [comment]
> @ScharfViktor I also added setter and getter for `$userWhoCreatedLastPublicShare` because other test code might need to be able to set that also.

Thank you))

**Comment by phil-davis** [comment]
@ScharfViktor I will make a commit id update in oCIS...

**Pull request merged**